pythainlp/corpus/words_th.txt

```diff
@@ -11810,6 +11811,7 @@
 โคเลสเตอรัล
 โคโลน่า
```

Comment:
does this mean `corona` or something else?

Reply:
> 
> 
> does this mean `corona` or something else? if yes, I guess it should be "โคโรน่า".

OK.